src/extension.ts

```diff
@@ -92,21 +103,18 @@ export function activate(context: ExtensionContext) {
         debug: {
             module: serverModule,
             transport: TransportKind.ipc,
-            options: debugOptions
-        }
+            options: debugOptions,
+        },
     };
 
     // Options to control the language client
     const clientOptions: LanguageClientOptions = {
         // Register the server for SLiM and Eidos files
-        documentSelector: [
-            { scheme: 'file', language: 'slim' },
-            { scheme: 'file', language: 'eidos' }
-        ],
+        documentSelector: [{ scheme: 'file', language: 'slim' }],
         synchronize: {
             // Notify the server about file changes to '.slim' files contained in the workspace
-            fileEvents: workspace.createFileSystemWatcher('**/*.{slim,eidos}')
-        }
+            fileEvents: workspace.createFileSystemWatcher('**/.clientrc'),
```

Comment:
i'm guessing that i'm missing something here, but previously the `workspace.createFileSystemWatcher` watched `**/*.{slim,eidos}`. Should this be watching `.slim` files?